lib/net/pool.js

```diff
@@ -1246,8 +1360,8 @@ class Pool extends EventEmitter {
         await this.handleBlockTxn(peer, packet);
         break;
       case packetTypes.CFILTER:
```

Comment:
nit: please move this up with the other messages related to the compact filters

Reply:
resolved